test/client/app/components/higherOrder/hasConfig.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import hasConfig from '../../../../../assets/app/components/higherOrder/hasConfig';
+
+const mock = React.createClass({
```

Comment:
Looking at this further, what you suggested won't work, React expects either a string representing a tag like 'div' or an actual React component, not just JSX. Providing a string produces an [unknown prop warning](https://facebook.github.io/react/warnings/unknown-prop.html)

Going to leave as it is for now!